src/distrs/tdist.jl

```diff
@@ -22,3 +14,27 @@ function tdistlogpdf(ν::T, x::T) where {T<:Real}
     νp12 = (ν + 1) / 2
     return loggamma(νp12) - (logπ + log(ν)) / 2 - loggamma(ν / 2) - νp12 * log1p(x^2 / ν)
 end
+function tdistcdf(ν::T, x::T) where {T<:Real}
+    if x < 0 && r < (0x1p-26 * x)^2
+       return -log(abs(x))*r + log(r)*muladd(r, .5, -1.) - logbeta(r/2, 1/2)
+    end
+    q = 0.5*beta_inc(r/2, 1/2, r/muladd(x, x, r))[1]
```

Comment:
This causes undesired promotions. We want e.g. that `Float32` inputs yield `Float32` results.